src/store/index.ts

```diff
@@ -86,7 +89,7 @@ export type AppReduxState = CombinedState<{
   [CURRENCY_VALUES_KEY]: CurrencyValuesState
   [COOKIES_REDUCER_ID]: Map<string, any>
   [ADDRESS_BOOK_REDUCER_ID]: AddressBookReducerMap
-  [CURRENT_SESSION_REDUCER_ID]: Map<string, any>
+  [CURRENT_SESSION_REDUCER_ID]: SerializedSessionState
```

Comment:
In your comment you say we are not using it and we should remove it.
But here we are using it?

Reply:
Yes but it's not consumed anywhere, it's just added there